### PR TITLE
Adds Coordinator module

### PR DESCRIPTION
**Pull request by rob5standingby**
## Why?
- Adds a `Coordinator` module to get user and organization info from the Contxt Coordinator API

## What changed?
- Added a new `Coordinator` module with THREE functions:
  - `getAllOrganizations`
  - `getOrganizationById`
  - `getUser`
- Added tests
- Updated the documentation
- **Note**: I'm not grabbing _all_ of the data returned on the `user` object from `getUser.` I've only pulled out the fields that I have an immediate need for. This can be further expanded later

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -71,6 +72,10 @@ class ContxtSdk {
 
     this.assets = new Assets(this, this._createRequest('facilities'));
     this.auth = this._createAuthSession(sessionType);
+    this.coordinator = new Coordinator(
```
Comment (greg5green) [inline src/index.js]: Can you add a test case for this?

**Comment by greg5green** [comment]
Wait, update the CHANGELOG too before merging 😂

**Comment by greg5green** [comment]
Or do it on master, whatever

File: src/index.js (hunk 1 above)
Comment (rob5standingby) [reply src/index.js]: Oops. Yup, on it now.

**Comment by rob5standingby** [comment]
Thanks, @greg5green ! I'll update the CHANGELOG on master

**Pull request merged**